account-integrations/safe/src/SafeWebAuthnPlugin.sol

```diff
@@ -130,8 +134,7 @@ contract SafeWebAuthnPlugin is BaseAccount {
             paramLen = abi.decode(userOp.signature[i:i+dataLen], (uint256));
             i += dataLen; // advance index
             // assign authenticatorData to sig splice
-            dataLen = paramLen;//((paramLen >> 5) + 1) << 5; // (round up to next slot)
-
+            dataLen = paramLen;
```

Comment:
In that linked suggestion, it moves around how the variables are used throughout. That is:
- `i` is always the index at the start of the data,
- `dataLen` is always used as the length of data to read
- `paramLen` is the length of the parameter

So assigning `dataLen` to be `paramLen` is hopefully helpful towards self-documenting code (and if redundant should be optimised out by the compiler anyway).
Using `paramLen` directly in the slice is different to how the slices are used everywhere else. So even though it would not error, it is less clear as a whole.

Reply:
Thanks for clarifying, that makes sense to me and agree that seems more clear